packages/grafana-e2e/src/components/queryEditors/testdata/index.ts

```diff
@@ -0,0 +1,12 @@
+import { componentFactory } from '../../../support';
+
+export const TestDataQueryEditor = componentFactory({
```

Comment:
I would call it pageObjectFactory because it's more in line with common e2e terms, what do you think?